server/templates/transcribe.html

```diff
@@ -238,6 +250,22 @@
             submitCore(false /* skipped */ );
         }
 
+        function changeDarkMode(element) {
+            if (element.checked) {
+                // Let there be dark.
+                document.body.style.backgroundColor = "#0f0f0f";
+                document.body.style.color = "white";
+                container.style.backgroundColor = "black";
+                container.style.color = "white";
+            } else{
```

Comment:
No space after "else"?
Are we animals?

Reply:
javascript is not my natural habitat, i'm used to auto format :/